wl/server/SyncThread.java

```diff
@@ -55,18 +55,24 @@ public void run() {
 				Runtime.getRuntime().exec(new String[] {
 				    "bash", "-c",
 				    "mysqldump -u" + Utils.config("databaseuser") + " -p" +
-				        Utils.config("databasepassword") + " " + Utils.config("addonsdatabase") +
-				        " > _addons_database_backup.sql"});
+				        Utils.config("databasepassword") + " -h" + Utils.config("databasehost") +
+				        " -P" + Utils.config("databaseport") + " " +
+				        Utils.config("addonsdatabase") + " > _addons_database_backup.sql"});
```

Comment:
Okay, I added the phase (0-3) and day of week (0-6) to the backup name. So it would take a full week with no maintainer noticing the problem until the working backup is lost.